sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/aio/_azure_configuration_client_async.py

```diff
@@ -113,8 +129,40 @@ def _create_appconfig_pipeline(self, **kwargs):
                 self._config.headers_policy,
                 self._config.user_agent_policy,
                 AppConfigRequestsCredentialsPolicy(self._config.credentials),
+                self._config.retry_policy,
                 SyncTokenPolicy(),
+                self._config.logging_policy,  # HTTP request/response log
+                DistributedTracingPolicy(**kwargs),
+                HttpLoggingPolicy(**kwargs),
+            ]
+
+        if not transport:
+            transport = AsyncioRequestsTransport(**kwargs)
+
+        return AsyncPipeline(
+            transport,
+            policies,
+        )
+
+    def _create_appconfig_aad_pipeline(self, base_url, credential, **kwargs):
+        transport = kwargs.get('transport')
+        policies = kwargs.get('policies')
+        if base_url.endswith('/'):
+            scope = base_url + '.default'
+        else:
+            scope = base_url + '/.default'
+
+        if policies is None:  # [] is a valid policy list
+            if hasattr(credential, "get_token"):
+                credential_policy = AsyncBearerTokenCredentialPolicy(credential, scope)
+            else:
+                raise TypeError("Unsupported credential: {}".format(credential))
```

Comment:
"Please provide an instance from azure-identity or a class that implement the "get_token" protocol"

TLDR: Explain how to recover from the error